core/utils/src/main/kotlin/edu/stanford/spezi/core/utils/foundation/knowledgesource/ComputedKnowledgeSource.kt

```diff
@@ -0,0 +1,12 @@
+package edu.stanford.spezi.core.utils.foundation.knowledgesource
+
+import edu.stanford.spezi.core.utils.foundation.RepositoryAnchor
+
+interface ComputedKnowledgeSource<
+    Anchor : RepositoryAnchor,
+    Value : Any,
+    Repository,
```

Comment:
I removed this one, since it is (hopefully) mainly used to differentiate computed knowledge sources to be accessing SendableSharedRepositories specifically, since it doesn't have the "Self" generic constraint in there, so we just omit it for now.